pkg/polylog/polyzero/logger_example_test.go

```diff
@@ -0,0 +1,37 @@
+package polyzero_test
+
+import (
+	"os"
+
+	"github.com/rs/zerolog"
+
+	"github.com/pokt-network/poktroll/pkg/polylog/polyzero"
+)
+
+func ExampleNewLogger() {
+	// Use whichever zerolog level you need.
+	level := zerolog.InfoLevel
+	// Specify the lowest level to log. I.e.: calls to level methods "lower"
+	// than this will be ignored.
+	levelOpt := polyzero.WithLevel(level)
+
+	// Construct logger.
+	// NB: adding WithOutput is optional; defaults to os.Stderr. It is needed
+	// here to print to stdout for testable example purposes.
+	logger := polyzero.NewLogger(levelOpt, polyzero.WithOutput(os.Stdout))
+
+	// All level methods are always available, but will only log if the level
+	// is enabled.
+	logger.Debug().Msg("debug message - should not see me")
+	logger.Info().Msgf("info message with %s", "formatting")
+	logger.Warn().Str("warn", "message").Send()
```

Comment:
So we use `Send` when this is not part of the log level but we want it to come thru anyway?

Reply:
The logger will always respect the logger level. `Send()` is for when you don't  have any log message text. Often you may just want to log some data / fields via `polylog.Event` methods like `#Str()` in the example.